src/index.html

```diff
@@ -1,12 +1,345 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="catalog-page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a href="#home" class="logo">
+          <img src="images/logo.png" alt="Logo">
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link is-active">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Samsung</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Смартфоны и телефоны</a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+                data-qa="nav-hover"
+              >Ноутбуки и компьютеры</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Гаджеты</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Планшеты</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Фото</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Видео</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="content">
+      <div class="content__container">
+        <div class="content__row">
+          <div class="card-container">
+            <div class="card card-container__card" data-qa="card">
```

Comment:
Also try to mix and delete unnecessary divs.(as in comment higher) More detail in video.
[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)
And add changes to styles according to changes in html.